server/app.ts

```diff
@@ -54,6 +54,16 @@ export async function main() {
 	await buildClients();
 	buildRateLimiter();
 
+	if (process.argv.includes("--validate")) {
+		try {
+			loadConfigFile();
```

Comment:
`loadConfigFile` doesn't throw exceptions when validation fails. This always succeeds and the process will exit with code 0.

Reply:
Just to clarify -- include a key in your config that doesn't exist.
```toml
foo = "bar"
```
When you run `yarn start --validate` the process will exit with code 0.

Reply:
Yea, I did that. I thought it wasn't working. I just wanted confirmation.

So what I should do is check if the schema for `.env` files doesn't match up with the `content` in `loadConfigFile` and throw a `log.error` if it does?

Reply:
You can verify it yourself. `echo $?` will show you the exit code of the previous command that was run.

See if `conf.validate` does what we want. It would be nice to also take into account the logic in `validateMail` too.

Reply:
I believe if we change this line `conf.validate({ allowed: "warn" });` to `conf.validate({ allowed: "strict" });` in `loadConfigFile` it will work. Checking now.